tests/test_enterprise/views/test_router_view.py

```diff
@@ -45,7 +45,8 @@ def setUp(self):
             'enterprise_course_run_enrollment_page',
             args=[self.enterprise_customer.uuid, self.course_run_id]
         ))
-        self.request.user.id = 1  # pylint: disable=invalid-name
+        # pylint: disable=invalid-name
```

Comment:
seriously? this is the change? 🤦‍♂

Reply:
Looks like it might still not be working @christopappas :(